ssconvert.rb

```diff
@@ -0,0 +1,36 @@
+require 'roo'
+
+def output_filename(sheet)
+  "inputs/efile_COAK_2016_#{sheet}.csv"
+end
+
+xlsx = Roo::Spreadsheet.open(ARGV[0])
+
+$stderr.puts "Converting #{ARGV[0]} to a series of CSV files..."
+
+xlsx.sheets.each do |sheet|
+  out = output_filename(sheet)
+
+  $stderr.puts "  -> #{out}"
+  file = xlsx.sheet(sheet).each_row_streaming.each_entry
+
+  headers = file.next
+
+  CSV.open(out, 'wb') do |csv|
+    csv << headers
+
+    loop do
+      row = file.next
+
+      # HACK: the datetimes seem to have a weird formatting string
+      # ("mm/dd/yyy") in the XLSX spreadsheet which has an extra "y" at the end.
+      row.each do |cell|
+        if cell.type == :date
+          cell.instance_variable_set(:@format, "yyyy-mm-dd")
```

Comment:
Yeah, the problem was that the cells came with a format string: `mm/dd/yyy`. This resulted in the formatted output for the cells being something like `06/13/17y`. This hack fixes the format by replacing the format string with one in a format which is more standard, so that the call to `cell.to_s` formats the cell's value to something like `2017-06-13`.